### PR TITLE
Uploading vision.ipynb as part of vision capabilities guide.

### PR DESCRIPTION
**Pull request by data-wombat**
Uploading vision capabilities Colab as part of DevRel CY

## Description of the change
Uploading executable Colab for vision capabilities guide (which is coming shortly).

## Motivation
Assigned the vision capabilities guide and wanted an executable Colab for demos

## Type of change
Documentation

## Checklist

- [ X] I have performed a self-review of my code.
- [ X] I have added detailed comments to my code where applicable.
- [ X] I have verified that my change does not break existing code.
- [X ] My PR is based on the latest changes of the main branch (if unsure, please run `git pull --rebase upstream main`).
- [ X] I am familiar with the [Google Style Guide](https://google.github.io/styleguide/) for the language I have coded in.
- [ X] I have read through the [Contributing Guide](https://github.com/google/generative-ai-docs/blob/main/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

File: site/en/gemini-api/docs/vision.ipynb [hunk 1]
```diff
@@ -0,0 +1,701 @@
+{
```
Comment (markmcd) [inline site/en/gemini-api/docs/vision.ipynb:1]: <blockquote>Images greater than 20MB cannot be handled</blockquote><p> </p><p>This isn't strictly true, is it? AIUI, it's requests of 20MB or more, so 5x4MB images would hit this limit.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (markmcd) [inline site/en/gemini-api/docs/vision.ipynb:1]: <blockquote>Individual frames are 258 tokens, and audio is 32 tokens per second. With metadata, each second of video becomes ~300 tokens, which means a 1M context window can fit about 55.5 minutes of video.&nbsp;</blockquote><p> </p><p>Precision nit: When you approximate <code>~300 tokens</code>, you're using 1  degree of precision, then you switch to 3 degrees of precision for <code>55.5 minutes</code>, which is  quite specific, and slightly off the full calculation.</p><p> </p><p>Either stick with the single-digit approximation all the way through (e.g. "~300 tokens, which fits just under an hour of video") or be precise the whole way through (e.g. <code>290 tokens, giving 57mins of video</code>).</p><p> </p><p>Personally I prefer the approximate version, as users likely still need to fit a prompt, instructions, and possibly other media in.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

**Review by markmcd — APPROVED** [review]
LG, I have some very small feedback, but approving now in case changes require syncing with devsite or something and they get done later.

Also FYI I pushed some commits here to fix the buttons & some other small things.

File: site/en/gemini-api/docs/vision.ipynb [hunk 2]
```diff
@@ -0,0 +1,706 @@
...
+      "source": [
+        "### Upload one or more locally stored image files\n",
+        "\n",
+        "Alternatively, you can upload your own files. You can download and use our drawings of [piranha-infested waters](https://storage.googleapis.com/generativeai-downloads/images/piranha.jpg) and a [firefighter with a cat](https://storage.googleapis.com/generativeai-downloads/images/firefighter.jpg). First, save these files to your local directory.\n",
+        "\n",
+        "Then click **Files** on the left sidebar. For each file, click the **Upload** button, then navigate to that file's location and upload it:\n",
+        "\n",
+        "<img width=400 src=\"https://ai.google.dev/tutorials/images/colab_upload.png\">\n",
+        "\n",
+        "When the combination of files and system instructions that you intend to send is larger than 20MB in size, use the File API to upload those files, as previously shown. Smaller files can instead be called locally from the Gemini API: Smaller files can be called locally from the Gemini API:\n"
```
Comment (pcoet) [inline site/en/gemini-api/docs/vision.ipynb:334]: It seems like the last sentence is redundant.

**Review by pcoet — APPROVED** [review]
Thanks for this!

File: site/en/gemini-api/docs/vision.ipynb (hunk 1 above)
Comment (data-wombat) [reply site/en/gemini-api/docs/vision.ipynb:1]: You're right, I'd fixed it later in the doc but not there. Fixed now.

Comment (data-wombat) [reply site/en/gemini-api/docs/vision.ipynb:1]: Done.

**Review by data-wombat — COMMENTED** [review]
Applied requested changes.

**Pull request merged**